_source/_posts/2020-06-22-deploy-dotnet-container-aws-fargate.md

```diff
@@ -3,112 +3,99 @@ disqus_thread_id: 8089923701
 
-These are the 3 components that containers hope to solve. Ease of management, size, and speed to start. There are others but for simplicity, we'll focus on those 3 as I feel (IMHO) they are the most relevant to business processes. Containers differ from a traditional virtual machine in one major way. They do not have an actual Operating System. This may come as a surprise to you, even if you might have dabbled with containers before. Containers instead have an abstraction of an operating system that provides hooks into the host operating system using standards and conventions supplied by the container system.
+To talk about containers, we have to talk about Virtual Machines. A virtual machine is a virtual representation of a computer all the way from its boot process to loading an entire operating system. This provides a lot of flexibility because you get to have fine-grained control over an entire virtual computer, but it has all the pitfalls of a whole computer as well. The Operating System has to be maintained, patched, and updated; disks have to be managed, and all manner of operational overhead goes into their care and feeding. Notably, they are also huge because an entire OS has to go on them. So when you start a Virtual Machine, you have to go through the entire boot sequence of a traditional computer.
 
-The de facto standard is Docker for all practical purposes, but just for clarity others do exist. This standardization is where the notion of a container comes from. In shipping there use to be no standardization and this made transporting items difficult because you have to figure out how to hold each item on a boat, a train, a truck. Containers provided a standard format for transport "you can put whatever you want inside the container so long as it opens this way, can bolt down this way, and are one of these dimensions." this made the logistical process of moving real-world item much more simplified. In this same way standardizing the way a container interacts with its host OS to delegate the responsibility for the execution of tasks simplifies management of the part that is important the application.
+These are the three problem areas that containers hope to solve; Ease of management, size, and speed to start. There are others, but for simplicity, we will focus on those three as I feel (IMHO) they are the most relevant to business processes. Containers differ from a traditional virtual machine in one significant way. They do not have an actual Operating System. This may come as a surprise to you, even if you might have dabbled with containers before. Containers instead have an abstraction of an operating system that provides hooks into the host operating system using standards and conventions supplied by the container system.
```

Comment:
```suggestion
These are the three problem areas that containers hope to solve: Ease of management, size, and speed to start. There are others, but for simplicity, we will focus on those three as I feel (IMHO) they are the most relevant to business processes. Containers differ from a traditional virtual machine in one significant way. They do not have an actual Operating System. This may come as a surprise to you, even if you might have dabbled with containers before. Containers instead have an abstraction of an operating system that provides hooks into the host operating system using standards and conventions supplied by the container system.
```
```suggestion
These are the three problem areas that containers hope to solve; Ease of management, size, and speed to start. There are others, but for simplicity, I will focus on those three, which I feel are the most relevant to business processes. Containers differ from traditional virtual machines in one significant way. They do not have an actual operating system. This may come as a surprise, even if you've dabbled with containers before. Instead, containers use an abstraction of an operating system that provides hooks into the host OS using standards and conventions supplied by the container system.
```
small tweaks for clarity, conciseness, and varying sentence syntax and word choice